### PR TITLE
Adding fixes for #4.

### LINKED ISSUE #4: Update Optional Comments In Kotlin Tutorials
https://chat.stackoverflow.com/transcript/message/37581345#37581345

### PR DESCRIPTION
**Pull request by AdamMc331**
Closes #4.

### CODE REVIEW

File: _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-1-new-project.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ There's a lot here, and I don't want this to be an ultra-detailed tutorial of th
 
 Now as great as this is, there's a few ways we can change the above code. Specifically:
 
-1. While Kotlin has nullability in the type system and shouldn't throw NPEs, it's not impossible. The `as` keyword forces a cast, and if one of our views fails to be cast correctly we could be in trouble. We'll fix that by using `as?` instead.
+1. While Kotlin has nullability in the type system and shouldn't throw NPEs, it's not impossible. The `as` keyword forces a cast, and if one of our views fails to be cast correctly we could be in trouble. We could fix that by using `as?` instead, but we'll keep it because if our views can't be cast properly, I would rather fail than hide the error.
```
Comment (TimCastelijns) [inline _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-1-new-project.md:95]: you mix the perspective of 'we' and 'I'

File: _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-2-recyclerview.md [hunk 2]
```diff
@@ -104,14 +104,14 @@ Here we create an inner class that is a ViewHolder, it takes in a View that is p
 	    }
 
 	    inner class TaskViewHolder(view: View?) : RecyclerView.ViewHolder(view) {
```
Comment (TimCastelijns) [inline _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-2-recyclerview.md:106]: I (kotlin noob) think you need the ?s after view

Comment (AdamMc331) [reply _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-2-recyclerview.md:106]: Yeah I pushed up another commit I realized it on reviewing too.

File: _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-3-adding-items.md [hunk 3]
```diff
@@ -74,7 +81,7 @@ And here is what the MainActivity looks like with our new code added:
 	    var adapter: TaskAdapter? = TaskAdapter()
 
 	    override fun onCreate(savedInstanceState: Bundle?) {
-	        ...
+	        // ...
```
Comment (TimCastelijns) [inline _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-3-adding-items.md:84]: at line 100 there is another `...`

Comment (AdamMc331) [reply _posts/2017-05-21-how-to-build-a-todo-list-in-kotlin-part-3-adding-items.md:84]: oops

**Pull request merged**